portfolio/src/main/webapp/calendar.js

```diff
@@ -49,11 +55,39 @@ function createNewCalendarEvent() {
     document.getElementById('event-end-time-warning').style.visibility =
```

Comment:
It can be simpler to use jquery to show/hide elements.

Sample code:
$('#event-end-time-warning').hide();
$('#event-end-time-warning').show();

That being said, using jquery is not a requirement. I see the benefits of learning vanilla javascript as well.

Reference:
https://api.jquery.com/show/

Reply:
Updated.